src/main/java/world/bentobox/aoneblock/AOneBlock.java

```diff
@@ -106,6 +103,24 @@ public void onEnable() {
         registerListener(holoListener);
     }
 
+    //Load some of Manager
+    public void loadData(boolean register) {
```

Comment:
I see this is called by src/main/java/world/bentobox/aoneblock/listeners/ItemsAdderListener.java but not sure why it needs the parameter.